src/components/header.tsx

```diff
@@ -197,31 +198,38 @@ const Header: React.FC<{
               <div
                 className={"navbar-item " + (burgerMenuIsOpen && "is-active")}
               >
-                <button
-                  role="button"
-                  className={
-                    "navbar-burger burger " + (burgerMenuIsOpen && "is-active")
-                  }
-                  aria-expanded="false"
-                  onClick={() => setBurgerMenuStatus(!burgerMenuIsOpen)}
-                  data-target="navbar"
-                >
-                  <img
-                    src={
-                      burgerMenuIsOpen
-                        ? require("../img/close.svg")
-                        : require("../img/menu.svg")
-                    }
-                    className="mr-3"
-                    width="16"
-                    height="12"
-                  />
-                  {burgerMenuIsOpen ? (
-                    <Trans>Close</Trans>
-                  ) : (
-                    <Trans>Menu</Trans>
+                <I18n>
+                  {({ i18n }) => (
+                    <button
+                      role="button"
+                      className={
+                        "navbar-burger burger " +
+                        (burgerMenuIsOpen && "is-active")
+                      }
+                      aria-expanded="false"
+                      aria-label={i18n._(burgerMenuIsOpen ? t`Close` : t`Open`)}
+                      onClick={() => setBurgerMenuStatus(!burgerMenuIsOpen)}
+                      data-target="navbar"
+                    >
+                      <img
+                        src={
+                          burgerMenuIsOpen
+                            ? require("../img/close.svg")
+                            : require("../img/menu.svg")
+                        }
+                        className="mr-3"
+                        width="16"
+                        height="12"
+                        alt=""
+                      />
+                      {burgerMenuIsOpen ? (
+                        <Trans>Close</Trans>
+                      ) : (
+                        <Trans>Menu</Trans>
+                      )}
+                    </button>
```

Comment:
I think the aria-label property here is redundant since we have text inside the button. [See here](https://www.aditus.io/aria/aria-label/#:~:text=If%20the%20element%20has%20text,label%20%2C%20as%20it%20is%20redundant.): 

> If the element has text content, you DON'T need to add aria-label , as it is redundant.

Reply:
oh right of course, thanks!